test/test_examples_3d.jl

```diff
@@ -56,6 +56,37 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
   end
 
 
+  @testset "elixir_euler_taylor_green_vortex.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_taylor_green_vortex.jl"),
+    l2   = [0.0015040212437368603, 0.2789329218326114, 0.27893292183261237, 0.2124964098560632, 0.32470622624447526],
+    linf = [0.012164089985932103, 1.4346175439010938, 1.4346175439009117, 1.242662948547226, 2.6937193352995052])
+  end
+
+  @testset "elixir_euler_shockcapturing.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_shockcapturing.jl"),
+    l2   = [0.01948803219944478, 0.013572377993230113, 0.013572377993230083, 0.013559504611364608, 0.07065364959065795],
+    linf = [0.19054185048160754, 0.23111606104899296, 0.23111606104899302, 0.20819549633732257, 0.7069189497358219])
+  end
+
+  @testset "elixir_euler_density_pulse.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_density_pulse.jl"),
+    l2   = [0.05719653010913493, 0.057196530109134995, 0.057196530109134974, 0.05719653010913503, 0.08579479516370271],
+    linf = [0.2741524961597339, 0.27415249615973347, 0.2741524961597339, 0.274152496159735, 0.41122874423960276])
+  end
+
+  @testset "elixir_euler_ec.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_ec.jl"),
+    l2   = [0.025279625678119623, 0.016639460251717957, 0.016639460251717947, 0.01663291354143171, 0.09140063001443112],
+    linf = [0.43997373040112175, 0.287404037174804, 0.2874040371748037, 0.2892796201004814, 1.5155258175016093])
+  end
+
+  @testset "elixir_mhd_alfven_wave.jl" begin
+  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave.jl"),
+    l2   = [0.0038738661034704125, 0.009037180073786493, 0.004170786122172154, 0.011603806159981751, 0.0062502092450648075, 0.009224722367531584, 0.003459461499238398, 0.011683811689607632, 0.0021983399070141674],
+    linf = [0.012633279141878506, 0.0326736523735531, 0.01292372050545837, 0.04445719221507506, 0.027990560478335258, 0.03454996296127533, 0.010220142796804721, 0.04499671504008128, 0.009849090317107453])
+  end
+
+
```

Comment:
Did you **copy** these reference values from `test_examples_3d_old.jl`? If yes, i.e., the reference values match *exactly* those of Taam, you should prefix each testset string with `taal-confirmed`. If not, this is something that should be fixed before merging.